integrations/transportation/tfl.js

```diff
@@ -0,0 +1,173 @@
+import moment from 'moment';
+import { ACTIVITY_TYPE_TRANSPORTATION, TRANSPORTATION_MODE_PUBLIC_TRANSPORT } from '../../definitions';
+import { HTTPError, ValidationError } from '../utils/errors';
+
+const LOGIN_PATH = 'https://account.tfl.gov.uk/api/login';
+const BASE_PATH = 'https://mobileapi.tfl.gov.uk'; // For fetching data
+
+
+function generateActivities(travelDays) {
+  const activities = [];
+  travelDays.forEach((day) => {
+    day.Journeys.forEach((journey) => {
+      if (journey.Charge <= 0) { // ignore topups
+        activities.push({
+          id: journey.StartTime, // a string that uniquely represents this activity
+          datetime: journey.StartTime, // a javascript Date object that represents the start of the activity
+          durationHours: journey.EndTime && moment(journey.EndTime).diff(moment(journey.StartTime)) / (60 * 60 * 1000), // a floating point that represents the duration of the activity in decimal hours
+          distanceKilometers: journey.TransactionType === 'Bus' ? 3.140 : null, // a floating point that represents the amount of kilometers traveled (https://www.whatdotheyknow.com/request/bus_passenger_journey_times)
```

Comment:
Let's set this to null and leave the estimation of the distance to the carbon model

Reply:
Ok